Tools/wasm/wasi.py

```diff
@@ -301,10 +294,10 @@ def main():
                         "--wasi preview2 "
                         # Enable thread support; causes use of preview1.
                         #"--wasm threads=y --wasi threads=y "
-                        # Map the checkout to / to load the stdlib from /Lib.
-                        "--dir {HOST_DIR}::{GUEST_DIR} "
-                        # Set PYTHONPATH to the sysconfig data.
-                        "--env {ENV_VAR_NAME}={ENV_VAR_VALUE}")
+                        # Map the installed lib folder to /lib so we find stdlib.
+                        # This uses Step 6 in getpath.py and the standard wasi
+                        # host_prefix of /
```

Comment:
[PEP 8 asks for](https://peps.python.org/pep-0008/#comments) complete sentences and proper punctuation.

```suggestion
                        # host_prefix of / .
```